src/application/crud/CreateVariantPage.tsx

```diff
@@ -44,14 +59,7 @@ export class CreateVariantPage extends Component<Props, State> {
             The first step to set up your mobile device is to add a variants.
             That will generate the code necessary to register UPS on your
             device. Learn more about variants in the{' '}
-            <a
-              href={
-                'https://docs.aerogear.org/aerogear/latest/getting-started-running.html#running'
-              }
-            >
-              {' '}
-              documentation{' '}
-            </a>
+            <a href={getLink('getting-started#running')}> documentation </a>
```

Comment:
I couldn't figure out where this mapped, in app it gives an empty link

Reply:
Probably you need to update your UPS server. It has been added here: https://github.com/aerogear/aerogear-unifiedpush-server/pull/1118/files#diff-8154e370aeedaebfe91fcc0487a5f659R3